tests/foreman/ui/test_computeresource.py

```diff
@@ -127,19 +129,25 @@ def test_create_libvirt_resource_different_names_negative(self, name):
         @Assert: A libvirt Compute Resource is not created
 
         """
+        include_list = [' ']
```

Comment:
@abalakh Yes you are definitely right that you are missing something.  I think you need to read all the history of this PR to get your answer yourself.  If you need more info, ping me in irc.